wolfssl/wolfcrypt/rsa.h

```diff
@@ -214,9 +214,9 @@ struct RsaKey {
     int   type;                               /* public or private */
     int   state;
     word32 dataLen;
-#ifdef WC_RSA_BLINDING
```

Comment:
Not sure it's ok to remove this WC_RSA_BLINDING guard for embedded targets that might be using RSA verify only.

Reply:
For verify-only targets this will still build because `rsa.h` always includes `random.h`, which defines `struct WC_RNG {`. But rng will be a dead code member of RsaKey.

Would recommend we guard this a different way, or always require WC_RSA_BLINDING with linuxkm. We already reject `WOLFSSL_RSA_VERIFY_ONLY` and `WOLFSSL_RSA_PUBLIC_ONLY` in `linuxkm/lkcapi_rsa_glue.c`.